observatory-platform/observatory/platform/docker/docker-compose.observatory.yml.jinja2

```diff
@@ -152,24 +152,37 @@ x-build: &build
 
 services:
   redis:
-    image: "redis:6.0.5"
+    image: redis:latest
     restart: always
     ports:
       - ${HOST_REDIS_PORT}:6379
     networks:
       - {{ docker_network_name }}
+    healthcheck:
+      test: ["CMD", "redis-cli", "ping"]
+      interval: 5s
+      timeout: 30s
+      retries: 50
 
   flower:
-    image: apache/airflow:1.10.14-python3.7
```

Comment:
I realise that the reason this image is used and not the built image is because flower doesn't need the Airflow DAGs to function nor does it need any of the other things we install in the Docker image, so to make it lighter weight the base Airflow image was used.

If the Airflow Docker name / version is set as a parameter somewhere then we could also set this dynamically and use the base Airflow Docker image.

Reply:
Reverting back from build to image.  Will template out as part of installation script work.